MessageBoxDisplayService.cs

```diff
@@ -22,11 +22,26 @@ public bool OnMemoryClear()
             }
             return isMemoryStored;
         }
+
+        public bool HistoryClearing()
+        {
+            string mesaj = "Doresti sa stergi istoricul?";
+            string titlu = "Stergere Istoric";
+            MessageBoxButtons butoane = MessageBoxButtons.YesNo;
+            DialogResult rezultat = MessageBox.Show(mesaj, titlu, butoane);
+            if (rezultat == DialogResult.Yes)
+            {
+                return true;
+            }
+            else return false;
+            
+        }
```

Comment:
De ce nu o metoda generica "PromptUser" cu niste parametri alesi corespunzator? In acest moment message box service-ul nu este generic, ci este "legat" (cuplat) de presenter.